tests/unit/test_contrib/test_pydantic/test_beanie_integration.py

```diff
@@ -1,13 +1,15 @@
-from typing import Optional
+from typing import TYPE_CHECKING, Optional, Type
 
 import beanie
-import pydantic
+
+if TYPE_CHECKING:
+    from pydantic import BaseModel
 
 from litestar.contrib.pydantic import PydanticDTO
 
 
-def test_generate_field_definitions_from_beanie_models() -> None:
-    class Category(pydantic.BaseModel):
+def test_generate_field_definitions_from_beanie_models(base_model: "Type[BaseModel]") -> None:
```

Comment:
nit: Maybe we could use `from __future__ import annotations` so that we can remove the stringified type annotation.

Reply:
Generally we want to test these without I think